packages/flutter_image/lib/network.dart

```diff
@@ -27,13 +27,10 @@ import 'package:flutter/widgets.dart';
 @immutable
 class NetworkImageWithRetry extends ImageProvider<NetworkImageWithRetry> {
   /// Creates an object that fetches the image at the given [url].
-  ///
-  /// The arguments must not be null.
-  const NetworkImageWithRetry(
-    this.url, {
-    this.scale = 1.0,
-    this.fetchStrategy = defaultFetchStrategy,
-  });
+  const NetworkImageWithRetry(this.url,
+      {this.scale = 1.0,
+      this.fetchStrategy = defaultFetchStrategy,
+      this.requestHeaders});
```

Comment:
Nit: please add the trailing comma so the more legible formatting is kept.

Reply:
Done.